repository/query/scripts/read_invitation_by_id_and_status_for_update.sql

```diff
@@ -0,0 +1,6 @@
+SELECT *
+FROM project_invitations
+WHERE
+    id = @id AND
+    status = @status
+FOR UPDATE
```

Comment:
In the next PR I added a different approach to appending "FOR UPDATE" to the queries.

https://github.com/jan-zabloudil/release-manager/pull/80#pullrequestreview-2064764277